server/index.js

```diff
@@ -115,14 +116,22 @@ async function handleAlert(location, alertType) {
       }
       braveAlerter.startAlertSession(alertInfo)
     } else {
+      currentSession.numberOfAlerts += 1 // increase the number of alerts for this session
```

Comment:
This rings bells for a race condition. What if there was another alert between the time when `currentSession` was read from the DB and the time that this is done writing it back again?

Generally speaking this won't be a problem because Stillness Alerts should be at least stillness_threshold apart from each other, but I could see a  case where there was a Duration and a Stillness Alert is very quick succession. 

The real solution to this would be a transaction. @tmackinnon , do you think that this is worth it considering the possibility of deadlocks?

If you decide against it, then the next best thing is to have the read and write statements as close together as possible to reduce the likelihood of something else messing with your data in between.

Reply:
@trisapeace I may be misunderstanding but isn't this already a transaction with `db.beginTransaction()` being called on line 78? 

In terms how this change will affect deadlocks, I can't say for sure but I think it's more valuable to keep this as a transaction vs the risk of deadlocks. If deadlocks increase, I think there are other areas of code that we could adjust to compensate this.

Reply:
I will think about this, and leave it for later work on this PR!

Reply:
You're right, Tara. This is already in a transaction. Thanks!